tests/e2e/cucumber/features/smoke/link.feature

```diff
@@ -65,27 +65,59 @@ Feature: link
     And "Alice" logs out
 
 
-  Scenario: public link for folder (by authenticated user)
+  Scenario: public link for folder and file (by authenticated user)
     Given "Admin" creates following user using API
       | id    |
       | Brian |
     And "Alice" logs in
     And "Alice" creates the following resources
       | resource     | type   |
       | folderPublic | folder |
+    And "Alice" creates the following resources
+      | resource         | type    | content   |
+      | shareToBrian.txt | txtFile | some text |
+      | shareToBrian.md  | mdFile  | readme    |
```

Comment:
maybe these files can go inside one folder, share the folder two Brian, create link shares of these files and Brain opens the links (expected to be inside a shared folder with file being selected - checked)